### PR TITLE
OPP Student Library - Basic UI

### PR DESCRIPTION
**Pull request by muskan2532150**
-  Create an app.rb file to serve as your console app entry point. It should have methods that do the following:

      - List all books.
      - List all people.
      - Create a person (teacher or student, not a plain Person).
      -  Create a book.
      -  Create a rental.
      -  List all rentals for a given person's id.

- In your main.rb define the entry point, this will be a method called main invoked at the end of your file. This method should 
    do the following:
    -  Present the user with a list of options to perform.
    -  Lets users choose an option.
    -  If needed, ask for parameters for the option.
    -  Have a way to quit the app.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,56 @@
...
+
+### Setup
+1. Clone the repo in your local machine:
+```bash
+https://github.com/muskan2532150/OOP-school-library.git
+```
+2. Open terminal and cd into project and install the dependencies
+```bash
+cd OOP-school-library
+npm install
```
Comment (mmsesay) [inline README.md]: - [ ] `npm install` doesn't work for ruby. Please refactor to `bundle install`

File: README.md [hunk 2]
```diff
@@ -1,2 +1,56 @@
...
+```
+
+3. run the command to open your index.html automatically in your brower:
+```bash
+npm run build
+npm start
+```
```
Comment (mmsesay) [inline README.md]: - [OPTIONAL] I don't think this step is vital for a ruby project

File: app.rb [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+  def list_rentals
+    if @rentals.length.zero?
+      puts 'No rental found for books'
+    else
+      puts ''
+      print 'ID of a person: '
+      id = gets.chomp
+      @rentals.each do |rental|
+        if rental.person.id.to_i == id.to_i
+          puts "Date: #{rental.date} Book: #{rental.book.title} by #{rental.book.author}"
```
Comment (mmsesay) [inline app.rb:82]: - [ ] Only the year of the rental date is been displayed. Please work on it to show the entire date entered by the user.

Your App Output |
:-------------------:|
Screen Shot 2022-11-11 at 7 34 58 PM |

Expected Output |
:-------------------:|
Screen Shot 2022-11-11 at 7 37 22 PM |

**Review by mmsesay — CHANGES_REQUESTED** [review]
Hi @muskan2532150, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
 - Good Readme :heavy_check_mark: 
 - No linter error :heavy_check_mark: 
 - Good PR title :heavy_check_mark: 
 - Good PR summary :heavy_check_mark:  


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by VICTOR-CHUKWU — APPROVED** [review]
Hi @muskan2532150 , 
# STATUS: APPROVED! 🥇 💯 
done

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- [ ] All linters passed

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**